content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+Here is the **setenv.sh** file content for your reference; however, you do not have to modify these configuration in this tutorial. 
+<br />
+
+```bash hideClipboard
+#!/bin/bash
+# Number of VMs to provision
+export NO_OF_VMS=3
+export VM_PREFIX="demo"
+export INSTALLER_TEMPLATE="palette-edge-template"
+
+#### DO NOT MODIFY BELOW HERE ####################
+# GOVC Properties
+export GOVC_URL="https://${vcenter_server}"     # Use HTTPS. For example, https://vcenter.company.com
+export GOVC_USERNAME="${vcenter_username}"
+export GOVC_PASSWORD="${vcenter_password}"
+export GOVC_INSECURE=1 #1 if insecure
+export GOVC_DATACENTER="${vcenter_datacenter}"
+export GOVC_DATASTORE="${vcenter_datastore}"
+export GOVC_NETWORK="${vcenter_network}"
+export GOVC_RESOURCE_POOL="${vcenter_resource_pool}"
+export GOVC_FOLDER="${vcenter_folder}"
+```
+
+The **setenv.sh** file customization is out of the scope of this tutorial. To change the number of VMs to provision, you must clone the [tutorials](https://github.com/spectrocloud/tutorials.git) repository and edit the **edge-native/vmware/clone_vm_template/setenv.sh** file, rebuild the image using these [local build steps](https://github.com/spectrocloud/tutorials/blob/main/docs/docker.md#local-builds) and use that image in the previous `docker run` command. 
+<br />
+
+
+# Deploy a Cluster
+After building the Edge artifacts and provisioning VMs as Edge hosts, the next step is to verify the host registration, create a cluster profile, and deploy a cluster. 
+<br />
+
+## Verify Host Registration
+Ideally, VMs should automatically register with Palette if you correctly embed the registration token in the ISO image. Log back into Palette to verify, and navigate to the left **Main Menu** > **Clusters** > **Edge Hosts** tab. You should see the VMs registered with Palette automatically.
+
+![A screenshot showing the VMs registered with Palette automatically. ](/tutorials/edge-native/clusters_edge_deploy-cluster_edge-hosts.png)
+
+
+If you do not see your Edge hosts registered with Palette automatically, you can register the hosts using the Edge host ID manually. Click on the **Add Edge Hosts** button, and paste the Edge host ID returned after provisioning the VMs in one of the previous steps.
+<br />
+
+## Create a Cluster Profile
+Switch to the **Default** project scope for creating a cluster profile.  
+<br />
+
+![A screenshot of Palette's Default scope.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_default-scope.png)
+
+<br />
+
+Select the **Profile** section in the left **Main Menu** to create a cluster profile comprising the core infrastructure layers and a manifest of a sample Kubernetes application, [Hello Universe](https://github.com/spectrocloud/hello-universe#hello-universe). 
```

Comment:
```suggestion
```
Let's remove this and combine with the next line. Also, the order is incorrect, mention the left main menu, then the profiles section.